neurodamus/node.py

```diff
@@ -1157,6 +1157,7 @@ def _reports_init(self, pop_offsets_alias):
             CoreConfig.write_spike_filename(file_name)
         else:
             # once all reports are created, we finalize the communicator for any reports
+            self._sonatareport_helper.set_max_buffer_size_hint(SimConfig.report_buffer_size)
```

Comment:
where was this before? We just had this function without using it?

Reply:
It was used in stand-alone CoreNEURON